types/kotlin2/UserProperties.kt

```diff
@@ -22,14 +22,18 @@ import im.vector.app.features.analytics.itf.VectorAnalyticsEvent
 // https://github.com/matrix-org/matrix-analytics-events/
 
 /**
- * The user properties to apply when identifying. These properties should all be
- * device independent.
+ * The user properties to apply when identifying. This is not an event
+ * definition. These properties must all be device independent.
  */
-data class Identify(
+data class UserProperties(
         /**
          * The selected messaging use case during the onboarding flow.
          */
         val ftueUseCaseSelection: FtueUseCaseSelection? = null,
+        /**
+         * Number of spaces (and sub-spaces) the user is joined to
+         */
+        val numSpaces: Int? = null,
 ) : VectorAnalyticsEvent {
```

Comment:
we'll need to drop extending from this interface as it'll attempt to enforce `getName`